tf-dataflow/tfproc/tfproc.py

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/env python
```

Comment:
This file should be renamed to tfdataflow? Where is the name `tfproc` coming from?

Reply:
tfproc was this thing's first name ("tensorflow processor"). We can rename it.

Reply:
I got confused with `Dataproc` offer by GCP. I think we either `tfdataflow` or `tfproc` works in this case, but I think we should make python package name and root folder name consistent.